intellihide.js

```diff
@@ -198,7 +199,7 @@ const Intellihide = new Lang.Class({
              * select a window in the secondary monitor.
              */
 
-            let monitorIndex = this._settings.get_int('preferred-monitor');
+            let monitorIndex = this._monitorIndex;
```

Comment:
This is not used, right? The tracked actor already bring information about the monitor. I guess you included for consistency, maybe we can skip it.